oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/RelatedSelectField/NoResultsMessage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Message, Grid, Button } from "semantic-ui-react";
+
+export const NoResultsMessage = ({ noResultsMessage, handleModal }) => {
+  return (
+    <Message>
+      <Message.Content>
+        <Grid columns={2} verticalAlign="middle">
+          <Grid.Column>
+            <p>{noResultsMessage}</p>
+          </Grid.Column>
+          <Grid.Column textAlign="right">
+            <Button
+              fluid
+              as="a"
+              color="green"
+              icon="search"
+              labelPosition="left"
+              content="Search External Database"
```

Comment:
This should be a customizable prop (defined by user of the RelatedSelectField). Users of the component may want the button to say e.g. "Search in DatabaseXYZ"